### PR TITLE
[RHCLOUD-23693] Configurable default bundles

### PR DESCRIPTION
**Pull request by Ellen-Yi-Dong**
Make list of default bundles to be configurable so we don't need to change code with the new bundle. 

Added new parameter within deployment/deploy.yml and added it to the env. 

Updated config/main.go to reflect the changes made in deploy.yml

Related JIRA ticket: [RHCLOUD-23693](https://issues.redhat.com/browse/RHCLOUD-23693)
# Platform Security

## Secure Coding Practices Checklist Link
- https://github.com/RedHatInsights/secure-coding-checklist 

## Secure Coding Checklist
- [ ] Input Validation

- [ ] Output Encoding

- [ ] Authentication and Password Management

- [ ] Session Management

- [ ] Access Control

- [ ] Cryptographic Practices

- [ ] Error Handling and Logging

- [ ] Data Protection

- [ ] Communication Security

- [ ] System Configuration

- [ ] Database Security

- [ ] File Management

- [ ] Memory Management

- [ ] General Coding Practices

### CODE REVIEW

File: deployment/deploy.yml [hunk 1]
```diff
@@ -190,3 +192,6 @@ parameters:
   name: RUN_BUNDLE_SYNC
   required: false
   value: 'false'
+- description: Application allow list for new bundles with skus to update the feature in the IT features API
+  name: BUNDLES_SKUS_ALLOW_LIST
+  value: ansible, smart_management_rods, rhoam, rhosak, openshift, rhel
```
Comment (lindgrenj6) [inline deployment/deploy.yml:197]: I think we just need to remove the spaces there - from what it was set to before I imagine it's splitting on the `","` character.

**Review by lindgrenj6 — COMMENTED** [review]
Just one tiny comment, otherwise looks good!

**Comment by lpichler** [comment]
@Ellen-Yi-Dong can also create those envs for eph,stage, production - @ abaiken did similar thing here https://gitlab.cee.redhat.com/service/app-interface/-/merge_requests/56993

when we need to add bundle we just need to create MR with this change in app-interface.

**Comment by Ellen-Yi-Dong** [comment]
I created the MR for the envs here: [](https://gitlab.cee.redhat.com/service/app-interface/-/merge_requests/58282)
Please let me know if anything should be added or changed! Thank you so much!!

**Comment by lpichler** [comment]
@Ellen-Yi-Dong can you rebase please ? 🙏

**Review by dagbay-rh — CHANGES_REQUESTED** [review]
marking as needs work for now so I don't lose track of it in my todo list

lgtm tho pending the changes we talked about (getting clowdapp synced with deploy)

**Review by dagbay-rh — APPROVED** [review]
:cowboy_hat_face:

**Pull request merged**